### PR TITLE
watch the ays repo using inotify

### PR DESCRIPTION
**Pull request by mie00**

### CODE REVIEW

**Comment by zaibon** [comment]
we also need to handle the templateRepo : https://github.com/Jumpscale/jumpscale_core8/blob/8.2.0/lib/JumpScale/baselib/atyourservice81/lib/TemplateRepo.py#L6
and the templates themself:
https://github.com/Jumpscale/jumpscale_core8/blob/8.2.0/lib/JumpScale/baselib/atyourservice81/lib/TemplateRepo.py#L202

File: lib/JumpScale/baselib/atyourservice81/lib/TemplateRepo.py [hunk 1]
```diff
@@ -1,7 +1,24 @@
...
-import os
+import inotify
+
+
+def get_root_template_repo_if_relevant(path):
+    template_repo = None
+    if 'templates' in path:
+        template_repo = path.split("templates")[0]
+    elif 'tests' in path:
+        template_repo = path.split("tests")[0]
```
Comment (zaibon) [inline lib/JumpScale/baselib/atyourservice81/lib/TemplateRepo.py]: Why do we match on `tests` ?

Comment (rkhamis) [inline lib/JumpScale/baselib/atyourservice81/lib/TemplateRepo.py]: Otherwise no testsuites would run

**Comment by rkhamis** [comment]
pyinotify is forked here: https://github.com/gigforks/PyInotify

**Pull request merged**